src/test/java/io/strimzi/kafka/bridge/http/ConsumerSubscriptionTest.java

```diff
@@ -275,6 +253,7 @@ void listConsumerSubscriptions(VertxTestContext context) throws InterruptedExcep
                         assertThat(response.body().getJsonArray("partitions").size(), is(2));
                         assertThat(response.body().getJsonArray("partitions").getJsonObject(0).getJsonArray(topic2).size(), is(4));
                         assertThat(response.body().getJsonArray("partitions").getJsonObject(1).getJsonArray(topic).size(), is(1));
+                        listSubscriptions.complete(true);
```

Comment:
try outside the `context.verify` and remove the `context.completeNow` it should be at the end.
Again, try to use the same pattern as `subscriptionConsumerDoesNotExistBecauseAnotherGroup` test.